### PR TITLE
Implement localStorage on Proposal Vote list

### PR DESCRIPTION
**Pull request by jcmelati**

### CODE REVIEW

File: front-end/src/hooks/authContext.js [hunk 1]
```diff
@@ -1,72 +1,87 @@
...
-        isLoggedIn,
-        login,
-        createAccount,
-        logout
-    };
+  return {
+    civic,
+    isGov,
+    isLoggedIn,
+    isLoggedInValue,
```
Comment (theblockstalk) [inline front-end/src/hooks/authContext.js:66]: why is this needed now? isLoggedIn() returns this

File: front-end/src/hooks/voteContext.js [hunk 2]
```diff
@@ -1,31 +1,57 @@
...
+    const proposalExists = proposals.find(x => x.proposalId === proposal.proposalId)
+    if (!proposalExists) {
+      const newProposals = [...proposals, proposal]
+      setProposals(newProposals)
+      setProposalsStorage(newProposals)
+    }
+  }
+
+  useEffect(() => {
+    if (authContext.isLoggedIn()) {
```
Comment (theblockstalk) [inline front-end/src/hooks/voteContext.js:28]: await authContext.isLoggedIn()

File: front-end/src/hooks/voteContext.js [hunk 3]
```diff
@@ -1,31 +1,57 @@
...
+      const proposalsStorage = getProposalsStorage();
+      if (proposalsStorage) {
+        setProposals(proposalsStorage);
+      }
+    }
+  }, [authContext.isLoggedInValue]);
```
Comment (theblockstalk) [inline front-end/src/hooks/voteContext.js:34]: authContext.isLoggedIn should be this maybe?

File: front-end/src/hooks/authContext.js (hunk 1 above)
Comment (jcmelati) [reply front-end/src/hooks/authContext.js:66]: isLoggedIn() only returns true or false
civic instance is necessary for the civic calls outside the context
I am also using isLoggedInValue state to update proposals on the voteContext

File: front-end/src/hooks/voteContext.js (hunk 3 above)
Comment (jcmelati) [reply front-end/src/hooks/voteContext.js:34]: authContext.isLoggedIn is a function, not a state
